packages/core/src/lib/GooglePlay.ts

```diff
@@ -58,4 +61,53 @@ export class GooglePlay {
     await this.gradleWrapper.executeGradleCommand(
         ['publishBundle', '--artifact-dir', filepath, '--track', track]);
   }
+
+  /**
+   * This calls the Google Play Console through the nodejs api and then calls bundles.list which
```

Comment:
I feel this doc is a little too conversational - do you mind changing it to something like "Connects to the Google Play Console, retrieves a list of all Android App Bundles for the given packageName and returns the max versionCode of those bundles. Considers both ChromeOS and Android releases".

Reply:
This is a bad habit of mine. Changed it to be a little less conversational.